### PR TITLE
Register

### PR DESCRIPTION
**Pull request by winny1212**
Set up email and password form for register page

### CODE REVIEW

File: src/views/register/Input.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (NatachaBartu) [inline src/views/register/Input.js:1]: Well done Winny!

File: src/views/register/Register.js [hunk 2]
```diff
@@ -1,11 +1,60 @@
...
+      <Paper className={classes.paper} elevation={3}>
+        <Avatar sx={{ margin: 1, backgroundColor: 'secondary.light' }}>
+          <LockIcon />
+        </Avatar>
+
+        <Typography component="h1" variant="h5">
+          {isSignup ? 'Sign up' : 'Sign in'}
+        </Typography>
+
+        <form>
```
Comment (NatachaBartu) [inline src/views/register/Register.js:32]: Good Form!

File: src/views/register/styles.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from '@mui/styles';
```
Comment (NatachaBartu) [inline src/views/register/styles.js:1]: need talk about the dependency....

Comment (a-sh-dev) [reply src/views/register/styles.js:1]: Hey @winny1212, the jsx structure seems good!!!

**Comment by a-sh-dev** [comment]
`import { TextField, Grid, InputAdornment, IconButton } from '@material-ui/core';`
Hey WInny, it seems that you're using the old material version.. anything with `@material-ui/core` is the old version. Thang needs to fix his work too. I'm afraid when we use both versions it will cause issues for deployment.

**Pull request merged**